xbmc/music/windows/GUIWindowMusicBase.cpp

```diff
@@ -1179,7 +1179,8 @@ bool CGUIWindowMusicBase::CanContainFilter(const std::string &strDirectory) cons
 void CGUIWindowMusicBase::OnInitWindow()
 {
   CGUIMediaWindow::OnInitWindow();
-  if (CMediaSettings::Get().GetMusicNeedsUpdate() == 35)
+  if (CMediaSettings::Get().GetMusicNeedsUpdate() == 53 && !g_application.IsMusicScanning() &&
```

Comment:
This looks wrong. I specifically changed that a short while ago because there were issues for Isengard. The condition shouldn't contain the checks for music scanning and library containing music because that check is done two lines further down.